### ISSUE TITLE
SQL: NOT NULL constrained failed: courses.semester

### ISSUE THREAD

**Issue by werli** [fknorr/studip-client#14]
After installing and initiating the directory as shown in the [installation guide](https://github.com/fknorr/studip-client#installation) calling `studip {update, download, sync}` gives me the following error.

Running on Arch Linux with Python 3.5 and the default `studip-client` configuration.

```
Using last sync directory /home/manuelneuer/some/random/dir  ...
Stud.IP user name: neuer01
Password: 
Synchronize Hauptseminar Real Life Security B. Sc.? ([Y]es, [n]o, [m]etadata only): 
Traceback (most recent call last):
  File "/usr/bin/studip", line 9, in <module>
    load_entry_point('studip-client==1.1', 'console_scripts', 'studip')()
  File "/usr/lib/python3.5/site-packages/studip_client-1.1-py3.5.egg/studip/application.py", line 258, in main
    app.run()
  File "/usr/lib/python3.5/site-packages/studip_client-1.1-py3.5.egg/studip/application.py", line 241, in run
    self.update_database()
  File "/usr/lib/python3.5/site-packages/studip_client-1.1-py3.5.egg/studip/application.py", line 173, in update_database
    self.session.update_metadata()
  File "/usr/lib/python3.5/site-packages/studip_client-1.1-py3.5.egg/studip/session.py", line 126, in update_metadata
    self.db.add_course(course)
  File "/usr/lib/python3.5/site-packages/studip_client-1.1-py3.5.egg/studip/database.py", line 168, in add_course
    type=course.type, sync=int(course.sync), expected_rows=0)
  File "/usr/lib/python3.5/site-packages/studip_client-1.1-py3.5.egg/studip/database.py", line 104, in query
    cursor.execute(sql, dict(**kwargs))
sqlite3.IntegrityError: NOT NULL constraint failed: courses.semester
```

**Comment by fknorr** [comment]
May have been fixed by a76ab66 .

**Comment by werli** [comment]
This is correct.
This was caused by setting a different home page in Stud.IP than the courses overview
